crates/admin/src/cluster_controller/grpc_svc_handler.rs

```diff
@@ -182,9 +182,11 @@ impl ClusterCtrlSvc for ClusterCtrlSvcHandler {
             }
             Ok(Snapshot {
                 snapshot_id,
+                log_id,
                 min_applied_lsn,
             }) => Ok(Response::new(CreatePartitionSnapshotResponse {
                 snapshot_id: snapshot_id.to_string(),
+                log_id: log_id.as_u32(),
```

Comment:
I think it's more idiomatic to use `into()` in rust
```suggestion
                log_id: log_id.into(),
```